db/migrate/20200824085207_create_maglev_sites.rb

```diff
@@ -1,6 +1,6 @@
-class CreateMaglevSites < ActiveRecord::Migration[6.0]
+class CreateMaglevSites < Maglev::Migration
```

Comment:
will a move to a custom migration subclass

1) Make rails g migration annoying in the future for maglev devs?
2) Any concerns about older migrations dealing with Migration version quirks as newer AR versions are released and this number is bumped in future migrations?

Reply:
good catch, you're absolutely right! Instead, I now inject a module in the classes where I need the primary key type.